docs/source/man.rst

```diff
@@ -308,84 +308,145 @@ As the network infrastructure changes, it might be necessary to delete some sour
+
+``qpc scan edit --name=myscan --sources network1sources network2sources``
+
+**TIP:** After editing a scan use the ``qpc scan show`` command to review those edits.
 
 Listing and Showing Scans
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``qpc scan list`` command returns the summary details for all executed scans. The output of this command includes the identifier, the source or sources, and the current state of the scan.
+The ``qpc scan list`` command returns the summary details for all created scan objects or all created scan objects of a certain type. The output of this command includes the identifier, the source or sources, and any options supplied by the user.
 
-**qpc scan list** **--type=** *(connect | inspect)* **--state=** *(created | pending | running | paused | canceled | completed | failed)*
+**qpc scan list** **--type=** *(connect | inspect)*
 
 ``--type=type``
 
   Optional. Filters the results by scan type. This value must be ``connect`` or ``inspect``. A scan of type ``connect`` is a scan that began the process of connecting to the defined systems in the sources, but did not transition into inspecting the contents of those systems. A scan of type ``inspect`` is a scan that moves into the inspection process.
 
-``--state=state``
+The ``qpc scan job`` command returns the the list of scan jobs or a single scan job associated with a scan object. The output of this command includes the scan job identifiers as well as the status of each job and the results.
 
-  Optional. Filters the results by scan state. This value must be ``created``, ``pending``, ``running``, ``paused``, ``canceled``, ``completed``, or ``failed``.
+**qpc scan job** --name** *scan_name* **--id=** *scan_job_identifier* **--status=** *(created | pending | running | paused | canceled | completed | failed)*
 
-The ``qpc scan show`` command is the same as the ``qpc scan list`` command, except that it returns summary details for a single specified scan. You can also use this command to display the results of a scan.
+``--name=name``
 
-**qpc scan show --id=** *scan_identifier* **[--results]**
+  Required. Contains the name of the scan object of which to display the scan jobs.
 
-``--id=scan_identifier``
+``--id=scan_job_identifier``
+
+  Optional. Contains the identifier of a specified scan job to show.
+
+``--status=status``
+
+  Optional. Filters the results by scan job state. This value must be ``created``, ``pending``, ``running``, ``paused``, ``canceled``, ``completed``, or ``failed``.
+
+The ``qpc scan show`` command is the same as the ``qpc scan list`` command, except that it returns summary details for a single specified scan object.
+
+**qpc scan show --name** *name*
+
+``--name=name``
+
+  Required. Contains the name of the scan object to display.
+
+The ``qpc scan status`` command returns the status details for a single specified scan job.
+
+**qpc scan status** **--id=** *scan_job_identifier*
```

Comment:
We should move this to the scanning section below.

Reply:
Done.